### PR TITLE
Laod test suite

### PR DESCRIPTION
**Pull request by Nitish1814**
changes
1- added time to report
2- report will be created under ./perfTestReport folder
3- every report will overwrite previous report

### CODE REVIEW

File: .github/workflows/load-test.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: load-test
 
 on:
+  push:
```
Comment (sonalgoyal) [inline .github/workflows/load-test.yml:4]: why on push?

File: common/core/src/main/java/zingg/common/core/block/Canopy.java [hunk 2]
```diff
@@ -16,6 +16,7 @@
 
 
 public class Canopy<R> implements Serializable {
+	private static final long serialVersionUID = -229533781044789499L;
```
Comment (sonalgoyal) [inline common/core/src/main/java/zingg/common/core/block/Canopy.java:19]: why are we adding it here? and this value in any case is not making sense.

File: perf_test_input.py [hunk 3]
```diff
@@ -7,13 +7,14 @@
 #load file config to test on
 config_65 = "examples/febrl120k/config.json"
```
Comment (sonalgoyal) [inline perf_test_input.py:8]: rename to febrl, febrl120k, ncVoter5m

**Review by sonalgoyal — CHANGES_REQUESTED** [review]
report name has date which means it is not getting overwritten?

see other comments.

File: perf_test_input.py (hunk 3 above)
Comment (Nitish1814) [reply perf_test_input.py]: done

**Pull request merged**